apps/desktop/src/lib/branch/SeriesHeader.svelte

```diff
@@ -98,11 +97,18 @@
 		sourceBranch && shouldCheck ? $forge?.checksMonitor(sourceBranch) : undefined
 	);
 
-	// Trigger refresh of pull request status and checks when branch(es) are pushed.
+	// Extra reference to avoid potential infinite loop.
+	let lastSeenPush: Date | undefined;
+
+	// Without lastSeenPush this code has code into an infinite loop, where lastPush seemingly
```

Comment:
`This code has code into an infinite loop`? :joy:

Reply:
I'm glad you read the comments! 🤗